src/java/us/kbase/mobu/tester/ModuleTester.java

```diff
@@ -229,14 +230,16 @@ public void logNextLine(String line, boolean isError) {
                         }
                     }).build();
             ModuleRunVersion runver = new ModuleRunVersion(
-                    new URL("https://fakefakefakefakefake.com"),
-                    new ModuleMethod("use_set_provenance.to_set_provenance_for_tests"),
-                    "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa", "0.0.0", "dev");
+                    new URL("https://localhost"),
+                    new ModuleMethod(moduleName + ".to_set_provenance_for_tests"),
+                    "local-docker-image", "local", "dev");
             final DockerMountPoints mounts = new DockerMountPoints(
                     Paths.get("/kb/module/work"), Paths.get("tmp"));
+            Map<String, String> localModuleToImage = new LinkedHashMap<>();
+            localModuleToImage.put(moduleName, imageName);
```

Comment:
I don't think method is necessary in this structure. And since it all exists in purely SDK code (not common execution part) we can change it later at any time. And I'm too lazy to it now anyway.